src/backend/base/langflow/components/agents/agent.py

```
@@ -268,3 +270,16 @@ async def update_build_config(
                         component_class, build_config, field_value, "model_name"
                     )
         return dotdict({k: v.to_dict() if hasattr(v, "to_dict") else v for k, v in build_config.items()})
+
+    async def _get_tools(self) -> list[Tool]:
+        component_toolkit = _get_component_toolkit()
+        tools_names = self._build_tools_names()
+        agent_description = self.get_tool_description()
+        # TODO: Agent Description Depreciated Feature to be removed
+        description = f"{agent_description}{tools_names}"
+        tools = component_toolkit(component=self).get_tools(
+            tool_name="Call_Agent", tool_description=description, callbacks=self.get_langchain_callbacks()
```

Pylint (3.3.7)

[convention] Line too long (106/100)

(C0301)